samples/tab-calendar/ts/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css" />
+    <title>Interact with calendar in Outlook from personal tab</title>
```

Comment:
This sample app shows how to interact with the user's calendar, including opening calendar items and composing meetings from personal tab in Outlook.

Reply:
Fixed